tiledb/sm/array/array_directory.cc

```diff
@@ -406,6 +411,10 @@ ArrayDirectory::load_commits_dir_uris_v12_or_higher(
       name = name.substr(0, name.size() - constants::write_file_suffix.size());
       fragment_uris.emplace_back(
           uri_.join_path(constants::array_fragments_dir_name).join_path(name));
+    } else if (stdx::string::ends_with(
```

Comment:
This loop is for `consolidated_uris`. Shouldn't we add this to the following loop for `commits_dir_uris`? Let me know if I misunderstood.